_attributes/common-attributes.adoc

```diff
@@ -15,6 +15,7 @@
 :op-system-base: RHEL
 :op-system-base-full: Red Hat Enterprise Linux (RHEL)
 :op-system-version: 9.x
+:op-system-version-9: 9
```

Comment:
```suggestion
:op-system-version-9: 9.0
```
Usually we write versions as **7.0, 8.0, 9.0**. But I wouldn't hold merge for this.